community/how_to_ask_on_discourse.md

```diff
@@ -1,4 +1,140 @@
 (how_to_ask_on_discourse)=
-# How to ask a question on Discourse
+# Asking Questions on the PyMC Discourse
+
+This document provides some advice for those asking questions on the [PyMC Discourse site](https://discourse.pymc.io/), the main channel for communication among the PyMC community.  If you have questions regarding modeling, PyMC/Aesara/arviz usage, or Bayesian modeling in general, please create a new post and someone is likely help you out.
+
+The items in this guide are designed to help maximize the chances that you good answers quickly.
+
+- {ref}`ask_discourse/search`
+- {ref}`ask_discourse/problem`
+- {ref}`ask_discourse/mwe`
+- {ref}`ask_discourse/title`
+- {ref}`ask_discourse/versions`
+- {ref}`ask_discourse/tags`
+- {ref}`ask_discourse/example`
+- {ref}`ask_discourse/summary`
+
+(ask_discourse/search)=
+## Search
+
+The PyMC Discourse is full of questions that have already received useful answers.  So definitely use the search function to see if anyone has asked questions that are similar to yours.  Try different combinations of keywords because past questions don't necessarily include {ref}`title`<a great title>.  You can also search disourse for topics by selecting relevant {ref}`tags` (or filter search results by tag).  You should also try searching {ref}`the PyMC documentation <pymc/>` because it covers both the PyMC API and common use-cases.  If you find something that is related but does not solve your specific problem, include links to these related pages and explain why the what you have found does not solve your problem.
```

Comment:
```suggestion
The PyMC Discourse is full of questions that have already received useful answers.  So definitely use the search function to see if anyone has asked questions that are similar to yours.  Try different combinations of keywords because past questions don't necessarily include {ref}`a great title <title>`.  You can also search discourse for topics by selecting relevant {ref}`tags` (or filter search results by tag).  You should also try searching {doc}`the PyMC documentation <pymc:index>` because it covers both the PyMC API and common use-cases.  If you find something that is related but does not solve your specific problem, include links to these related pages and explain why the what you have found does not solve your problem.
```